cmds_test.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/alecthomas/kong"
+	"github.com/matryer/is"
+)
+
+var buildtimeVarTests = []struct {
+	name string
+
+	buildHost, buildPSK string
+	envHost, envPSK     string
+	cliHost, cliPSK     string
+
+	wantHost, wantPSK string
+}{
+	{"none", "", "", "", "", "", "", "", ""},
+	{"build only", "example.com", "1234", "", "", "", "", "example.com", "1234"},
+	{"env only", "", "", "example.com", "1234", "", "", "example.com", "1234"},
+	{"cli only", "", "", "", "", "example.com", "1234", "example.com", "1234"},
+	{"build+env", "example.com", "1234", "example2.com", "9876", "", "", "example2.com", "9876"},
+	{"build+cli", "example.com", "1234", "", "", "example2.com", "9876", "example2.com", "9876"},
```

Comment:
feels like there should also be one for build+env_cli (sorry for the ocd remark).

Reply:
That would be testing that kong is doing what it should be doing. If that test fails, it would not be because of the offscreen code. I think. Maybe. Do you agree perhaps?

Reply:
yeah, i think so.